services/scripts/testy-snapshot.sh

```diff
@@ -70,6 +79,9 @@ main() {
     fi
 
     # Validate database. Update the snapshot status on success/failure.
+    aws ec2 create-tags --resources "$_snapshot_id" "$_volume_id" \
```

Comment:
`_snapshot_id` -> `_backup_snapshot_id`
`_volume_id` -> `_backup_snapshot_id`

Reply:
Fixed in [6afcb40](https://github.com/wiredtiger/testy/pull/50/commits/6afcb408ff6a150f84ade3514b4c2e12302ecbb4).